README.md

```diff
@@ -2,6 +2,21 @@ A Julia library to handle projections on manifolds. This is useful for minimizin
 
 Currently, the sphere `{x ∈ K^n, ||x|| = r}` and the Stiefel manifold `{X ∈ K^{n × m}, X'*X = I}` as well as independent copies of these manifolds are supported.
 
+The projections implemented are retract and project_tangent (both are also available inplace `!`).
+```
+retract(M::Manifold, x) = retract!(M, copy(x))
```

Comment:
No need to say how `retract` is implemented to the user. Maybe just say (after the "example" section) how to define a new manifold, which would function as an API documentation? Taking for instance the example of the "partial sphere" you had.